Recruit-Training/Advanced-Recruit-Training/src/LogFileReader.cpp

```diff
@@ -57,15 +57,23 @@ bool LogFileReader::readAll(const QString& fileName)
  * that the conversion from string to double is sucessful.*/
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
+
     QStringList sections = line.split(BATDATA_DELIMITER);
+    if(sections.length() != 3){
```

Comment:
Standard bracket convention in the style guide has the brackets starting on a new line